<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programmable set of Commands</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>131</width>
     <height>221</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>History box</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Display</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopMButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardMbutton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftMButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightMButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="first">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>1st Command</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="second">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>2nd Command</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="third">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>3rd Command</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fourth">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>4th Command</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiting to start a test</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>168</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="threeButton">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>3s</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fiveButton">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>5s</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose delay between commands</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sendCommand">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to Kilobots</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset test</string>
   </property>
  </widget>
  <widget class="QLabel" name="sendLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeWinButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Window</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton_2</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fiveButton</sender>
   <signal>clicked()</signal>
   <receiver>threeButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threeButton</sender>
   <signal>clicked()</signal>
   <receiver>fiveButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
